README.adoc

```diff
@@ -248,18 +255,18 @@ https://www.ibm.com/support/knowledgecenter/was_beta/com.ibm.websphere.wlp.nd.mu
 
 == Enabling distributed tracing
 
-While the OpenTracing feature (`opentracing-1.0`) found inside Open Liberty enables tracing of all JAX-RS methods by default,
-MicroProfile OpenTracing allows you to control and customize these traces. Using the `@Traced` annotation
-you can enable and disable tracing of particular methods. A custom `Tracer` object can also be injected
-for creating and customizing spans.
+The OpenTracing feature (`opentracing-1.0`) found inside Open Liberty enables tracing of all JAX-RS
+methods by default, but you can use MicroProfile OpenTracing to control and customize these traces.
+Use the `@Traced` annotation to enable and disable tracing of particular methods. You can also inject
+a custom `Tracer` object to create and customize spans.
 
 
-=== Enabling distributed tracing with no code instrumentation
+=== Enabling distributed tracing without code instrumentation
 
-Since tracing of all JAX-RS methods is enabled by default, enabling MicroProfile OpenTracing and the
-Zipkin user feature in the `server.xml` file is enough to see some basic traces in Zipkin.
+Because tracing of all JAX-RS methods is enabled by default, you need only to enable MicroProfile
+OpenTracing and the Zipkin user feature in the `server.xml` file to see some basic traces in Zipkin.
 
-Both of these features have already been enabled in `inventory/src/main/liberty/config/server.xml`
+Both of these features are already been enabled in the `inventory/src/main/liberty/config/server.xml`
```

Comment:
We should remove "been" so it reads, "...features are already enabled in..."

Reply:
Good catch, fixed.